src/wp-includes/class-wp-theme-json.php

```diff
@@ -3101,6 +3101,53 @@ protected static function get_metadata_boolean( $data, $path, $default_value = f
 		return $default_value;
 	}
 
+	/**
+	 * An internal method to get the block nodes from a theme.json file.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param array $theme_json The theme.json converted to an array.
+	 * @return array The block nodes in theme.json.
```

Comment:
Given this is a separate function with the only purpose to return the node paths, it would make more sense to return an array of strings, not an array of associative node arrays with only the path in it.

But we may want to reconsider the approach, per my overarching comment.